src/components/InputTodo.js

```diff
@@ -0,0 +1,49 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+//  eslint-disable-next-line
+class InputTodo extends Component {
+  constructor() {
+    super();
+    this.state = {
+      title: '',
+    };
+  }
+
+  onChange = (e) => {
+    this.setState({
+      [e.target.name]: e.target.value,
+    });
+  };
+
+  handleSubmit = (e) => {
```

Comment:
I think it will be better if you don't enforce the need for memorizing the variable context. You can learn more about clean code best practices in this [link](https://devinduct.com/blogpost/22/javascript-clean-code-best-practices)